.github/workflows/ci.yml

```
@@ -56,6 +56,9 @@ jobs:
         sudo rm /etc/apt/sources.list.d/* && sudo dpkg --clear-avail # Speed up installation and get rid of unwanted lists
         sudo apt-get update
         sudo apt-get install -y --no-install-recommends libxmu-dev libxi-dev libgl-dev libglu1-mesa-dev libgles2-mesa-dev libwayland-dev libxkbcommon-dev libegl1-mesa-dev libosmesa6 mesa-utils libglvnd-dev
+        # Disable VM sound card because it is unreliable.
+        echo 'pcm.!default { type plug slave.pcm "null" }' >> /tmp/asound.conf
+        sudo sh -c 'cat /tmp/asound.conf >> /etc/asound.conf'
```

```suggestion
        sudo echo 'pcm.!default { type plug slave.pcm "null" }' >> /etc/asound.conf
```

---

That would execute "echo" as root, and then redirect as the normal user.

```suggestion
        sudo sh -c 'echo "pcm.!default { type plug slave.pcm \"null\" }"  >> /etc/asound.conf'
```